--- a/spotdl/web/routes.py
+++ b/spotdl/web/routes.py
@@ -41,6 +41,7 @@
     Handle the home page request.
     """
     return templates.TemplateResponse(
+        request,
         name="home.html.j2",
         context={"request": request, "__version__": __version__},
     )
@@ -52,6 +53,7 @@
     Handle the search input.
     """
     return templates.TemplateResponse(
+        request,
         name="search.html.j2",
         context={"request": request, "__version__": __version__, "search_term": q},
     )
@@ -63,6 +65,7 @@
     Handle the downloads page request.
     """
     return templates.TemplateResponse(
+        request,
         name="downloads.html.j2",
         context={
             "request": request,